src/model.js

```diff
@@ -1283,16 +1283,21 @@ ${associationOwner._getAssociationDebugList()}`);
           const references = currentAttribute.references;
           if (currentAttribute.references) {
             const database = this.sequelize.config.database;
-            const schema = this.sequelize.config.schema;
+            const schema = tableName.schema;
```

Comment:
shouldn't this default to `this.sequelize.config.schema` if it's not set? Maybe something to change in `Model.getTableName()`